src/constructors/createGlobalStyle.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import React from 'react'
+import PropTypes from 'prop-types'
+import hashStr from '../vendor/glamor/hash'
+import StyleSheet, { CONTEXT_KEY } from '../models/StyleSheet'
+import ServerStyleSheet from '../models/ServerStyleSheet'
+import type { Interpolation, Stringifier } from '../types'
+import {
+  CHANNEL,
+  CHANNEL_NEXT,
+  CONTEXT_CHANNEL_SHAPE,
+} from '../models/ThemeProvider'
+
+export default (stringifyRules: Stringifier, css: Function) => {
+  const createGlobalStyle = (
+    strings: Array<string>,
+    ...interpolations: Array<Interpolation>
```

Comment:
You'll want to run `css()` ASAP on this input so that it runs only once (see `keyframes.js`)